src/main/java/frc/robot/constants/RobotConstants.java

```diff
@@ -41,7 +51,8 @@
     public static class VisionConstants {
         public double DISTANCE_CALCULATION_A_COEFFICIENT;
         public double DISTANCE_CALCULATION_B_COEFFICIENT;
-        // Offsets are measured from the robot's center of rotation to the limelight position.
+        // Offsets are measured from the robot's center of rotation to the limelight
+        // position.
```

Comment:
// Offsets are measured from the robot's center of rotation to the limelight
        // position.
same line